R/tetraplot.R

```diff
@@ -186,7 +186,7 @@ tetraplot <- function(tcsdata, theta = 45, phi = 10, perspective = FALSE,
   }
 
 
-  par(mar = margin, pty = "s", new = TRUE)
+  par(pty = "s", new = TRUE)
```

Comment:
The `par(pty = "s")` is necessary here and cannot be replaced by `asp = 1`?
If so, it'd be helpful to add a comment to explain why.

Reply:
Nope, just didn't search thoroughly enough...

`asp = 1` produces better results anyway.